packages/manager/apps/ips/src/pages/onboarding/index.tsx

```diff
@@ -174,32 +209,54 @@ export default function Onboarding() {
                     </tr>
                   </thead>
                   <tbody>
-                    {IP_OPTIONS_ROWS.map((row, index) => (
+                    {IP_OPTIONS_ROWS.map((row) => (
                       <tr
```

Comment:
if would be easier to read if you create a separate component for each line of the table:

```
<IpOptionRow key={row.feature} {...row} />
```